js/samples/04.ai.f.vision.cardGazer/README.md

```diff
@@ -1,17 +1,18 @@
-# Microsoft Teams Vision Enabled Bot : Card Master
+# Microsoft Teams Vision Enabled Bot : Card Gazer
```

Comment:
Could you also update the "samples" section of the getting start docs with the updated name